### DISCUSSION TITLE
使用大模型翻译，并且开启长段落自动换行，程序的翻译规则是？

### DISCUSSION THREAD

**Discussion by corwin1949** [immersive-translate/immersive-translate#2044] [Help] [answered]
### 概述 | Summary

你好，请问：

使用大模型翻译，并且开启长段落自动换行，程序的翻译逻辑是：
1. 翻译原内容后，再切分换行
2. 切分内容，再翻译

如果是 2 的话，我就不开启长段落自动换行了，大模型翻译有上下文会好点。

---

并且想问一下，翻译一篇长文章的话，使用大模型是每段内容分别独立翻译的吗

**Comment by theowenyoung** [comment, marked as answer]
逻辑是2.


默认情况下 每段内容是独立翻译的，你可以在翻译服务的设置中修改 每次发送的段落数，但是部分大模型对多段落的情况并不能很好的遵守。

**Reply by corwin1949** [reply]
感谢您的解答。这对我很有帮助。
